ui/src/app/components/runtime-configuration-panel/customize-panel.tsx

```diff
@@ -253,13 +252,13 @@ export const CustomizePanel = ({
                 <TooltipTrigger
                   content={`Cannot change the zone when an environment or persistent disk exists. 
                 If you would like to change zones, please make sure that you first delete both your environment and your persistent disk`}
-                  disabled={canChangeZone}
+                  disabled={!gcePersistentDisk}
                 >
                   <div>
                     <Dropdown
                       id='runtime-zone'
                       appendTo='self'
-                      disabled={disableControls || !canChangeZone}
+                      disabled={disableControls || !!gcePersistentDisk}
```

Comment:
sign flipped

Reply:
disabled={disableControls || !canChangeZone}
before my change this was equivalent to:
`disabled={disableControls || !(!currentRuntime || !gcePersistentDisk)}`
now it equals:
disabled={{disableControls || !( ~~!currentRuntime ||~~ !gcePersistentDisk)}} 
which is is equivalent to:
`disabled={{disableControls || !(!gcePersistentDisk)}}`
`disabled={{disableControls || !!gcePersistentDisk}}`

Would it be more readable to keep changeZone? That way I could add a comment explain what is happening.
`changeZone = !gcePersistentDisk`

Reply:
No worries. I updated it. Please let me know if you think the comment should be tweaked.